base/cvd/cuttlefish/host/libs/web/android_build_api.cpp

```diff
@@ -423,6 +424,8 @@ Result<std::string> AndroidBuildApi::DownloadTargetFile(
     }
     // Fallback to download from AB.
   }
+  CF_EXPECT(EnsureDirectoryExists(
+      std::string(cf_android::base::Dirname(target_filepath))));
```

Comment:
`cf_android` is the namespace after the import script rewrites the source, it's `android::base` here.